### DISCUSSION TITLE
Hiding icons on diagram key

### DISCUSSION THREAD

**Discussion by renannprado** [structurizr/ui#26] [Q&A]
Hello 👋 

I'm creating some diagrams, and I sometimes use icons in it. So far so good.

Due to the usage of icons and different colors and tags, I end up with the below diagram keys.

<img width="904" alt="image" src="https://user-images.githubusercontent.com/2359543/233006107-08377ee2-bcbb-4836-975d-7ab8740bb6a5.png">

However what I really need in the diagram key sometimes is just the explanation of the different colors, not much more than that.

Is it possible to manipulate the diagrams key somehow so that I can make it look like how I want it to look like?

**Comment by simonbrowndotje** [comment]
The icons are an integral part of the notation, so they're included in the diagram key and can't be removed I'm afraid.
